website/content/docs/nia/cli/cli-overview.mdx

```diff
@@ -1,14 +1,16 @@
 ---
 layout: docs
 page_title: Consul-Terraform-Sync CLI
-sidebar_title: CLI
+sidebar_title: Overview
 description: >-
   How to use the Consul-Terraform-Sync CLI
 ---
 
 # Consul-Terraform-Sync Command (CLI)
 
-Consul-Terraform-Sync is controlled via an easy to use command-line interface (CLI). Consul-Terraform-Sync is only a single command-line application: `consul-terraform-sync`.
+Consul-Terraform-Sync is controlled via an easy to use command-line interface (CLI). Consul-Terraform-Sync is only a single command-line application: `consul-terraform-sync`. The Consul-Terraform-Sync's main process can be run using additional flags. It also has a subcommands that can be used to interact with Consul-Terraform-Sync when running in daemon mode. The complete list of subcommands is in the navigation to the left. Both the main process and subcommands will return a non-zero exit status on error.
+
+## Main Process
```

Comment:
I hadn't thought about this until your call out here! What do you think about server/client? Where the CLI is the client

Reply:
Thanks for the suggestion! I like the accuracy of the server/client language. My only hesitation is if users will confuse the terminology with the Consul server/client dynamic, which sounds similar but is very different 🤔. I'll chat with you offline so that we can discuss and if other people have input, they can chime in. Thanks!

Reply:
I think it will still work as long as that section doesn't mention the consul server or the local agent at all... if it is only about CTS it should read clear enough I think.